Vizzuality/half-earth-v3
# Fix/sidebar safari

## Fix sidebar on safari, Change AOI headline
### Description
The counter position was messing with the safari sidebar. Now with this new position it works on both browsers
Also, the AOI headline has been updated
### Testing instructions
-With Safari: Go to the home page. You should see the '1' counter number on the correct place and no horizontal scrolling

- Go to one AOI page and the AOI headline should be updated

### Feature relevant tickets

https://vizzuality.atlassian.net/browse/HE-300?atlOrigin=eyJpIjoiZjljM2YyMDZlYTgxNDM5ZTk0Njk2OGQ2MDk1MDQ5NDMiLCJwIjoiaiJ9
https://vizzuality.atlassian.net/browse/HE-302?atlOrigin=eyJpIjoiYzI3ZDdhNjRkMjEwNGQ2NWEwZmFiYWVlZmE3MmUzY2QiLCJwIjoiaiJ9
